### DISCUSSION TITLE
Update error

### DISCUSSION THREAD

**Discussion by luckyaigit** [nagadomi/nunif#124] [General]
![1715852523720](https://github.com/nagadomi/nunif/assets/127811938/14b9cac0-731a-4909-8f44-af17715bf735)

FileNotFoundError: [Errno 2] No such file or directory
urllib.error.URLError: <urlopen error [Errno 2] No such file or directory>
Error

**Comment by nagadomi** [comment]
I am guessing this is a problem with your network env (VPN etc).
Also, since you have posted another issue, so I assume this issue has already resolved.
If you still have this problem, post an issue.

**Reply by luckyaigit** [reply]
Hi, nagadomi. Yes, this problem is due to the network, please help me with another post, that issue happened just today. Thank you very much!

**Comment by diverswan9** [comment]
I had a different but related Update error dealing with 'numpy'.

"ImportError: numpy.core.multiarray failed to import"

 After running 'update.bat' something about numpy.V2.0.0 not being compatible after it got installed through the update. I ran the 'gui' and it refused to load. Tried it 'run as administrator' and still nothing. Fortunately, I had a 'non-updated' version on another PC that had an older version of numpy, numpy-1.26.4. So I copied it to the one not working. (first I deleted V2.0.0 all numpy folders in, C:\Users\Admin\Desktop\nunif-windows\python\Lib\site-packages) The 'gui' worked with the older version of 'numpy'.

P.S. - I am not a 'coder' so this seemed like the easiest solution for me.

**Reply by nagadomi** [reply]
@diverswan9 
I think that is a problem that has already fixed today in https://github.com/nagadomi/nunif/issues/135
